src/app.js

```diff
@@ -330,5 +302,18 @@ export default function (app) {
 
     return element
   }
+
+  function event(type) {
+    var args = Array.prototype.slice.call(arguments, 1);
+    (subscriptions[type] || []).forEach(function(cb) { cb.apply(null, args) });
+    (subscriptions["*"] || []).forEach(function(cb) { cb.apply(null, type) });
```

Comment:
At one point I removed this, then I thought about the Chrome Dev tools [some](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd) [frameworks](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi) have to inspect apps written with them. If we get an tool like that this will be useful. Can always add it later anyways.